student.rb

```diff
@@ -1,9 +1,14 @@
 require './person'
+require './classroom'
+
+attr_accessor :classroom
 
 class Student < Person
   def initialize(classroom, age, name = 'Unknown', parent_permission: true)
     super(age, name, parent_permission)
+
     @classroom = classroom
+    classroom.student.push(self) unless classroom.student.include? self
```

Comment:
The app is not able to start as a result of the error shown below regarding the attribute_accessor. Kindly make sure that all errors are fixed and the app is working fine just as shown in the video before submit for another review. :+1: 

![Screenshot from 2023-06-22 23-36-11](https://github.com/joyapisi/school_library/assets/65233860/6846d659-ddd7-436d-8011-bfc2e5f1b87a)